### PR TITLE
Added social media sharing buttons.

### PR DESCRIPTION
**Pull request by Logan1x**
fixes: #12

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-www.nothingprivate.ml
```
Comment (gautamkrishnar) [inline CNAME]: No need to remove this file!

Comment (Logan1x) [inline CNAME]: Let me check i think its done mistakenly

**Comment by gautamkrishnar** [comment]
@Logan1x Thanks for taking your time to contribute... 👍

**Comment by tagawa** [comment]
Yeah, good job @Logan1x!

**Comment by Logan1x** [comment]
Thanks @gautamkrishnar and @tagawa for help.

**Pull request merged**